README.md

```
@@ -1,9 +1,1509 @@
+
+### Installation
+
+```bash
+npm install
+npm run build
+npm run dev
+```
+
+Visit `http://localhost:8000`
+
+## Project Structure
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)